spec/rails_helper.rb

```diff
@@ -7,6 +7,7 @@
 require "rspec/rails"
 require "capybara/rspec"
 require "capybara-screenshot/rspec"
+require "database_cleaner"
```

Comment:
Same as above, it blew up for me on a clean rake ci of master until I added this.  If it didn't on travis or for you, there's something non-deterministic.